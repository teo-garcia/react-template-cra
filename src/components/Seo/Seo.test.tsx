import { render } from '@testing-library/react';
import Seo from './Seo';

describe('<Seo /> tests', function () {
  test('Should match with the component snapshot', function () {
    const { asFragment } = render(
      <Seo title="Home" description="This is the homepage" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
