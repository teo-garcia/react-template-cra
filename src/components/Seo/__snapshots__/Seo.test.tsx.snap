// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Seo /> tests Should match with the component snapshot 1`] = `
<DocumentFragment>
  <title>
    Home
  </title>
  <meta
    content="This is the homepage"
    name="description"
  />
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    content=""
    property="og:title"
  />
  <meta
    content=""
    property="og:description"
  />
  <meta
    content=""
    property="og:description"
  />
  <meta
    content=""
    property="og:image"
  />
</DocumentFragment>
`;
